server/opts_test.go

```diff
@@ -1155,6 +1155,39 @@ func TestPanic(t *testing.T) {
 	}
 }
 
+func TestPingIntervalOld(t *testing.T) {
+	conf := createConfFile(t, []byte(`ping_interval: 5`))
+	defer os.Remove(conf)
+	opts := &Options{}
+	err := opts.ProcessConfigFile(conf)
+	if err == nil {
+		t.Fatalf("expected an error")
+	}
+	if err, ok := err.(*processConfigErr); !ok {
+		t.Fatalf("expected an error of type processConfigErr")
+	} else if len(err.warnings) != 1 {
```

Comment:
I meant for all of them. Again, see each t.Fatalf() as it there was a return. If there is a return, there is no need for a `else`.

Reply:
without the else here would be out of scope and not typed

Reply:
Ok, but you could have done:
```
err, ok := err.(*processConfigErr)
if !ok {
}
if len(err.warnings)..
```